owtf/scripts/ssl/reporter.py

```diff
@@ -8,15 +8,18 @@
 import traceback
 
 from template import RESULTS
-from owtf.utils.file import FileOperations
 
 
 if not os.path.isfile(sys.argv[1]):
     sys.exit(1)
 
 data = None
 try:
-    data = FileOperations.open(json.loads(str(sys.argv[1])))
+    # Opening the json document generated by ssl-labs
+    with open(sys.argv[1]) as f:
+        data = json.load(f)
```

Comment:
it should be `json.loads()` instead of `json.load`

Reply:
Thank You for reviewing.
`json.load()` can deserialize a file itself be passed directly in a file pointer, where `json.loads()` throws a `TypeError: expected string or buffer` error.

Reply:
@viyatb 
Thank You for reviewing.

json.loads() throws a TypeError: expected string or buffer error when a file pointer is directly passed to it, hence I used json.load() it can deserialize a file itself.
Let me know if I can help you with anything else.

Thanks